asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/metadata/PublishingStrategy.java

```diff
@@ -0,0 +1,11 @@
+package org.sahli.asciidoc.confluence.publisher.client.metadata;
+
+/**
+ * @author Laurent Verbruggen
+ */
+public enum PublishingStrategy {
```

Comment:
It doesn't feel right to me to have this class in the metadata package. I would put it in the client package what do you think?

Reply:
I agree. Fixed in bf5f0568a1c110f2e388e6ad23a8c1823f4023cc